client/index.jsx

```diff
@@ -38,4 +38,4 @@ class App extends React.Component {
   }
 }
 
-ReactDOM.render(<App />, document.getElementById('app'));
+ReactDOM.render(<App />, document.getElementById('photos'));
```

Comment:
We should make sure we all have this consistent, I altered mine to render to reviews, yours to photos as you have here, and Ahmed's to reservations

Reply:
Yep -- it's like that on my proxy. For the single component (this repo), I just have the one id.